src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -24,13 +24,13 @@ public class AddCommand extends Command {
             + "Parameters: "
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
-            + PREFIX_EMAIL + "EMAIL "
+            + PREFIX_CYCLE + "CYCLE "
             + PREFIX_ADDRESS + "ADDRESS "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
-            + PREFIX_EMAIL + "johnd@example.com "
+            + PREFIX_CYCLE + "johnd@example.com "
```

Comment:
Perhaps you could change the sample data as well? Something like the following
```suggestion
            + PREFIX_CYCLE + "Summer "
```

I saw this in some other parts of the code as well!